miio/vacuum.py

```diff
@@ -747,6 +790,29 @@ def set_waterflow(self, waterflow: WaterFlow):
         """Set water flow setting."""
         return self.send("set_water_box_custom_mode", [waterflow.value])
 
+    @command()
+    def mop_mode(self) -> Optional[MopMode]:
+        """Get mop mode setting."""
+        try:
+            return MopMode(self.send("get_mop_mode")[0])
+        except ValueError:
+            return None
```

Comment:
Please add a logger warning for unknown values, that way it doesn't go unnoticed.